### PR TITLE
[Platforms] Refactor openvino code

### PR DESCRIPTION
**Pull request by ji-huazhong**
Part of https://github.com/vllm-project/vllm/issues/9268

cc @youkaichao

### CODE REVIEW

File: vllm/platforms/openvino.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
 from typing import TYPE_CHECKING
 
+import openvino as ov
+import openvino.properties.hint as hints
 import torch
 
 import vllm.envs as envs
 from vllm.logger import init_logger
+from vllm.utils import GiB_bytes
```
Comment (youkaichao) [inline vllm/platforms/openvino.py:9]: I think you need to import it lazily inside the function

Comment (statelesshz) [reply vllm/platforms/openvino.py]: fixed in [4e5c594](https://github.com/vllm-project/vllm/pull/10573/commits/4e5c594644bdd8584fd8ebf83a83d55908ada5ae)

Comment (statelesshz) [reply vllm/platforms/openvino.py]: PTAL@youkaichao

**Review by youkaichao — APPROVED** [review]
thanks for the fix!

File: vllm/platforms/openvino.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
 from typing import TYPE_CHECKING
 
+import openvino as ov
+import openvino.properties.hint as hints
```
Comment (Isotr0py) [inline vllm/platforms/openvino.py:4]: I noticed these import broke the kernel tests: https://buildkite.com/vllm/ci-aws/builds/11712#01935994-0554-41aa-bad1-9d253bdef82f/194-278

**Pull request merged**